src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2905,6 +2905,42 @@ namespace winrt::TerminalApp::implementation
     }
     CATCH_LOG();
 
+    // Method Description:
+    // - This method is called when the user clicks the "export message history" button
+    //   in the query palette
+    // Arguments:
+    // - text - the text to copy to the clipboard
+    // Return Value:
+    // - <none>
+    void TerminalPage::_OnPasteToClipboardRequested(const IInspectable& /*sender*/, const winrt::hstring& text)
```

Comment:
I did not want to copy over the `_openClipboard` code/logic over to the ExtensionPalette (there are enough copies of that code already with one being in TermControl and one being in TerminalPage), so I figured we could have TerminalPage handle the request